docs/technical_reference/flowsheets/electrodialysis_1stack.rst

```diff
@@ -0,0 +1,107 @@
+
+.. figure:: ../../_static/flowsheets/edfs.png
+    :width: 500
+    :align: center
+
+    Figure 1. Flowsheet diagram: one-stack ED system
+
+The electrodialysis 1D block is set up with the following configuration arguments:
+
+.. code-block::
+
+   m.fs.EDstack = Electrodialysis1D(
+        property_package=m.fs.properties,
+        operation_mode="Constant_Voltage",
+        finite_elements=20,
+    )
+
+These configurations enable the electrodialysis unit to use a flowsheet-unified property package, set a constant stack voltage, and adopt a favorable number of finite elements for 1-dimensional simulation and solving.
+
+
+Documentation for unit models from WaterTAP: 
+    * `Electrodialysis_1D <https://watertap.readthedocs.io/en/latest/technical_reference/unit_models/electrodialysis_1D.html>`_
+Documentation for unit models from IDAES: 
+    * `Feed block <https://idaes-pse.readthedocs.io/en/latest/reference_guides/model_libraries/generic/unit_models/feed.html>`_
+    * `Separator <https://idaes-pse.readthedocs.io/en/latest/reference_guides/model_libraries/generic/unit_models/separator.html>`_
+    * `Product block <https://idaes-pse.readthedocs.io/en/latest/reference_guides/model_libraries/generic/unit_models/product.html>`_
+Documentation for the property model:
+    * `Multi-Component Aqueous Solution (MCAS) Property Package <https://watertap.readthedocs.io/en/latest/technical_reference/property_models/mc_aq_sol.html>`_
+
+Degrees of Freedom
+------------------
+The number of degrees of freedom (DOF) is associated with the number of fixed variables (parameters) determined by the purpose of the modeling case. We implemented two modeling cases in this flowsheet: (1) the prediction of desalination outcome (salinity of the product water and saline disposal) and (2) the optimization of key decision variables in system design.  In the first case, DOF is set to zero by fixing all initial conditions of the feed solution fluid and definite ED stack parameters. All fixed values are presented in the section to follow.  In the second case, the values of those chosen to be the decision variables in the optimization are unfixed. The DOF number is therefore the number of decision variables. In this example, the decision variables are
+
+    * stack voltage applied
+    * ED cell pair number 
+
+In the given optimization case, the objective function is to minimize the levelized cost of water, which can be represented by the following equation
+where :math:`Q` represents volumetric flow, :math:`f_{crf}` represents capital recovery factor
+:math:`C_{cap,tot}` represents total capital cost, :math:`C_{op,tot}` represents total operating cost, and
+:math:`f_{util}` represents the utilization factor:
+
+    .. math::
+
+        LCOW_{Q} = \frac{f_{crf}   C_{cap,tot} + C_{op,tot}}{f_{util} Q}
+
+
+The product water salinity is set to 1 :math:`g L^{-1}` (from a feed salinity of 9.9 :math:`g L^{-1}`).
```

Comment:
I think this should be moved from the DOF section to the end of the Implementation section